src/lib/consensus/proof_of_stake.ml

```diff
@@ -1335,13 +1364,15 @@ let next_proposal now (state : Consensus_state.value) ~local_state ~keypair
         ~ledger_hash:epoch_data.ledger.hash ~logger
     in
     let rec find_winning_slot slot =
-      if UInt32.of_int (Epoch.Slot.to_int slot) >= Constants.Epoch.size then
-        None
-      else
+      if UInt32.of_int (Epoch.Slot.to_int slot) >= Constants.Epoch.size then (
+        printf !"\ndone\n%!" ; None )
```

Comment:
Promote these printfs to logs or remove them.

Reply:
Woops, should have `git add -p` that file. This was leftover from debugging. They are too noisy, so I will kill them.